build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val root =
   project
     .in(file("."))
     .settings(moduleName := "cloud-storage-extension")
-    .settings(version := "1.0.0")
+    .settings(version := "1.1.0")
```

Comment:
Don't forget also to add a reproducibleBuild plugin, if you're going to release with Release Droid

Reply:
Ahh yes, thanks for reminding. I am going to add it in the next pr.